CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+* Fix linting issues
```

Comment:
I don't think this really needs to go in the changelog as it won't affect consumers of the gem (we hope).